src/pages/workspace/companyCards/assignCard/TransactionStartDateStep.tsx

```diff
@@ -55,11 +56,13 @@ function TransactionStartDateStep() {
     };
 
     const submit = () => {
+        const date90DaysBack = DateUtils.extractDate(subDays(new Date(), 90).toString());
+
         CompanyCards.setAssignCardStepAndData({
             currentStep: CONST.COMPANY_CARD.STEP.CONFIRMATION,
             data: {
                 dateOption: dateOptionSelected,
-                startDate,
+                startDate: dateOptionSelected === CONST.COMPANY_CARD.TRANSACTION_START_DATE_OPTIONS.FROM_BEGINNING ? date90DaysBack : startDate,
```

Comment:
@VickyStash Why do we set date90DaysBack here?

Reply:
@DylanDylann Please see this [discussion](https://callstack-hq.slack.com/archives/C05LX9D6E07/p1730793847260029?thread_ts=1730403357.940489&cid=C05LX9D6E07)

Reply:
Answered here: https://expensify.slack.com/archives/C05LX9D6E07/p1730793847260029?thread_ts=1730403357.940489&cid=C05LX9D6E07